README.md

```diff
@@ -36,18 +39,26 @@ The following controls are bound in-game, for gameplay and testing.
 | E | Interact with test cubes |
 | Tab | Toggle scoreboard |
 
-## Contributions
+#### Give us feedback
+We have released the GDK for Unreal this early in development because we want your feedback. Please come and talk to us about the software and the documentation via: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) - [GitHub issues in this repository](https://github.com/spatialos/UnrealGDK/issues).
 
-**Feedback**<br/>
-Please give us your feedback! Please see the Unreal GDK [feedback section](https://github.com/improbable/UnrealGDK#give-us-feedback) for more information.
+#### Where to get the GDK and example projects
+The GDK and its starter projects are available on GitHub.
+* [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
+* [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
+* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
+* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
+* [The SpatialOS Unreal Engine](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)
```

Comment:
This doesn't feel like it's in the same category as the others. Maybe move it up so it's just after the GDK link?